agent/bench-scripts/pbench-uperf

```diff
@@ -499,19 +499,16 @@ for protocol in `echo $protocols | sed -e s/,/" "/g`; do
 	done
 done
 
-# Start the tool meisters on each registered local/remote host
-pbench-tool-meister-start "${tool_group}"
-if [[ ${?} != 0 ]]; then
-	error_log "[${script_name}]: failed to start the tool meisters."
-	exit 1
-fi
-
-pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir --sysinfo=$sysinfo beg
-
-pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir beg
+if [[ $postprocess_only == "n" ]]; then
```

Comment:
We also fix this command to properly support `--postprocess-only` mode where the tools are also re-post-processed.